extensions/wikia/Insights/specials/UnconvertedInfoboxesPage.class.php

```diff
@@ -128,6 +129,11 @@ public function reallyDoQuery( $limit = false, $offset = false ) {
 	 * @return bool
 	 */
 	public static function isTitleWithNonportableInfobox( $titleText, $contentText ) {
+		// ignore docs pages
+		if ( strpos( $titleText, '/doc' ) ) {
```

Comment:
Just a minor edge-case consideration, but this will match `/doc` anywhere in the sting, not just at the end, so Template:Infoboc/doctor. I think it would be better to have an equivalent of `$title->isSubpage() && $title->getSubpageText() === 'doc'` to be precise about the actual goal.